src/datanode/src/instance.rs

```diff
@@ -330,6 +334,10 @@ impl Instance {
         self.procedure_manager
             .start()
             .context(StartProcedureManagerSnafu)?;
+        let _ = self
```

Comment:
```suggestion
        self
```

Reply:
It will be make a warning about `unused Result` if i remove let bind.